rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -40,6 +40,11 @@ def __init__(self, *args, **kwargs):
         TorchRLModule.__init__(self, *args, **kwargs)
         PPORLModuleBase.__init__(self, *args, **kwargs)
 
+        if self._is_discrete:
```

Comment:
We need to remove this stuff, once the other PR gets merged first.

Reply:
Yep.